thrusters:
  surge:
    channel: 1
    max_pwm: 1900
    min_pwm: 1100
    topic_name: control/thruster/surge

  heave:
    channel: 2
    max_pwm: 1900
    min_pwm: 1100
    topic_name: control/thruster/heave

servos:
  surge_port:
    channel: 2
    max_pwm: 2500
    min_pwm: 800
    topic_name: desired_servo_angle
    joint_name: surge_port_joint


  